Game.js

```diff
@@ -0,0 +1,77 @@
+const Board = require("./Board.js");
+const HumanPlayer = require("./HumanPlayer.js");
+const ComputerPlayer = require("./ComputerPlayer.js");
+const { hangManPics } = require("./hangmanPics.js");
+class Game {
+    constructor(players){
```

Comment:
You need a isGameOver method here that checks the game conditions to see if its over.